server.py

```diff
@@ -243,6 +244,36 @@ def do_get_response(tx_id):
         return jsonify({}), 404
 
 
+@app.route('/responses/delete-old', methods=['POST'])
+def delete_old_responses():
+    """Deletes responses that are older than the number of days set in config
+    Config use is a compromise for safety in case incorrect parameters are passed.
+    """
+    try:
+        response_retention_days = int(settings.RESPONSE_RETENTION_DAYS)
+
```

Comment:
might be worth raising an error at this point if settings.RESPONSE_RETENTION_DAYS is not set or is 0. Especially as the default value isn't set on that setting.

Reply:
It raises a 500 each time the end point is called if it is not set . The default value was deliberately left off so that a 500 would be generated if there was no positive action to set it. We discussed a minimum value for the RESPONSE_RETENTION_DAYS, but since it must be set as an enviroment variable then this was protection enough. Setting to zero would delete all records up to the previous midnight.